host/README.md

```diff
@@ -17,13 +17,13 @@ Download the latest  [Release package](https://github.com/instana/otel-dc/releas
 
 1) Download the installation package:
 ```bash
-wget https://github.com/instana/otel-dc/releases/download/v1.0.1/otel-dc-host-0.2.2.tar
+wget https://github.com/instana/otel-dc/releases/download/v1.0.3/otel-dc-host-0.2.3.tar
```

Comment:
should be:
wget https://github.com/instana/otel-dc/releases/download/v1.0.2/otel-dc-rdb-0.5.4.tar

Reply:
If we are going to create a new release v1.0.3, I think the path of rdb dc image will also be changed.